modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/updatePage.js

```diff
@@ -0,0 +1,70 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = `<?xml version="1.0" encoding="UTF-8"?>
+                <page xmlns="urn:enonic:xp:model:1.0">
+                  <display-name i18n="key.display-name">Virtual Page</display-name>
+                  <description i18n="key.description">My Page Description</description>
+                  <form>
+                    <input type="Double" name="pause">
+                      <label i18n="key1.label">Pause parameter</label>
+                      <immutable>false</immutable>
+                      <indexed>false</indexed>
+                      <occurrences minimum="0" maximum="1"/>
+                      <help-text i18n="key1.help-text"/>
+                    </input>
+                  </form>
+                  <regions>
+                    <region name="header"/>
+                    <region name="main"/>
+                    <region name="footer"/>
+                  </regions>
+                </page>
+                `
+
+// BEGIN
+// Update virtual page.
+var result = schemaLib.updateComponent({
+    key: 'myapp:mypage',
+    type: 'PAGE',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```